### DISCUSSION TITLE
How to use multi tenant with hierarchical structure?

### DISCUSSION THREAD

**Discussion by syedmubashirhussain** [Finbuckle/Finbuckle.MultiTenant#728] [Q&A]
Hi,
I'm looking into how to use multi-tenant with a hierarchical structure, Please share any examples. Also currently I'm using   Finbuckle.MultiTenant.AspNetCore (6.6.1) version.

Thanks

**Comment by AndrewTriesToCode** [comment]
There is currently no built in support. You can implement your own `IMultiTenantStore` that has this functionality if you need it.
